.github/workflows/quicktest.yml

```diff
@@ -76,10 +76,23 @@ jobs:
         if: ${{ matrix.phpcs_version == 'dev-master' }}
         run: composer lint -- --checkstyle
 
-      - name: Run the unit tests - PHP 5.4 - 8.0
-        if: ${{ matrix.php < '8.1' && matrix.php != 'latest' }}
+      - name: Run the unit tests without code coverage - PHP 5.4 - 8.0
+        if: ${{ matrix.php == '5.4' && github.ref_name != 'develop' }}
         run: composer run-tests
 
-      - name: Run the unit tests - PHP >= 8.1
-        if: ${{ matrix.php >= '8.1' || matrix.php == 'latest' }}
+      # Until PHPCS supports PHPUnit 9, we cannot run code coverage on PHP 8.0+, so run it on PHP 5.4 and 7.4.
+      - name: Run the unit tests with code coverage - PHP 5.4 - 8.0
```

Comment:
```suggestion
      - name: Run the unit tests with code coverage - PHP 5.4 - 7.4
```

Reply:
Ah, the `5.4 - 8.0` is not about the fact that code coverage is being run, but about the command used for PHPUnit, which works for PHP 5.4 - 8.0 ;-)